src/settings/Settings.js

```diff
@@ -130,6 +130,12 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "showHiddenEventsInTimeline": {
+        isFeature: true,
```

Comment:
technically this isn't a feature, it's a setting. Would recommend applying the same treatment as `enableWidgetScreenshots` to this.

Reply:
Ah okay, fixed!